subprojects/docs/src/docs/release/notes.md

```diff
@@ -2,6 +2,40 @@
 
 Here are the new features introduced in this Gradle release.
 
+### JUnit Platform and JUnit Jupiter Engine (a.k.a. JUnit 5) support
```

Comment:
I'd also expected documentation in the user guide here: https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_test

Reply:
I added a few lines to this doc. However, there's not much to write there because we don't support tagging and filtering yet.

Reply:
I added a few lines to this doc. However, there's not much to write there because we don't support tagging and filtering yet.